R/process_data.R

```diff
@@ -335,6 +335,21 @@ process_scenario_data <- function(data, start_year, end_year, sectors, technolog
   return(data_processed)
 }
 
+#' Process data of type indicated by function name
+#'
+#' @inheritParams process_carbon_data
```

Comment:
this is not entirely correct, if you are using inheritParams you need to inherit from a function where the arguments are described

Reply:
you use inhertigParams to inherit the parameter definitions form another function that has the paramters defined (in concrete another fucntion that has data, start_year and end_year defined). You cannot inherit data here, since it is specific to process_carbon_data. The other 2 params you could e.g. inherit from : process_production_data